index.html

```diff
@@ -57,25 +57,235 @@ <h2>you!</h2>
+                <div class="about-myself-letsconnect">
+                    <h2>LET'S CONNECT</h2>
+                </div>
+                <div class="about-myself-sociallinks">
+                    <ul class="about-myself-sociallists">
+                        <li> <a href="https://twitter.com/DannyDotcoder" target="_blank" rel="noreferrer noopener">
+                                <img src="img/Vector.png" alt="vector">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="https://www.linkedin.com/in/daniel-shdeed-832b03115/" target="_blank"
+                                rel="noreferrer noopener">
+                                <img src="img/Linkedinicon.png" alt="linkedinicon">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#" target="_blank" rel="noreferrer noopener">
+                                <img src="img/Vector(1).png" alt="vector1">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="https://github.com/Danieldotcomcoder" target="_blank" rel="noreferrer noopener">
+                                <img src="img/Vector(2).png" alt="vector2">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#" target="_blank" rel="noreferrer noopener">
+                                <img src="img/Vector(3).png" alt="vector3">
+                            </a>
+                        </li>
+                    </ul>
+                </div>
+
+            </div>
+            <div class="about-myself-getmyresume">
+                <a href="#" rel="noreferrer noopener">Get my resume</a>
+            </div>
         </div>
-    </div>
+        <div class="skill-list-section">
+            <ul class="skill-list-categories">
+
+                <li class="skill-languages-list">
+                    <div class="skill-language-text">
+                        <a href="#" rel="noreferrer noopener">
+                            <h2>Languages</h2>
+                        </a>
+                    </div>
+                    <a href="#" rel="noreferrer noopener"><img src="img/Frame9.png" alt="frame9"></a>
+                    <a href="#" rel="noreferrer noopener"><img src="img/Frame10.png" alt="frame10"></a>
+                    <a href="#" rel="noreferrer noopener"><img src="img/Frame11.png" alt="frame11"></a>
+                </li>
+                <li class="skill-framework">
+                    <a href="#" rel="noreferrer noopener">
+                        <h2>Frameworks</h2>
+                    </a>
+                </li>
+                <li class="skill-skill">
+                    <a href="#" rel="noreferrer noopener">
+                        <h2>Skills</h2>
+                    </a>
+                </li>
```

Comment:
Please follow the design for `Languages`, `Frameworks`, and `Skills`. They should have arrows as can be seen in the design.